utility/src/main/java/org/oppia/android/util/datetime/BUILD.bazel

```diff
@@ -0,0 +1,19 @@
+"""
+General purposes utilities for Android accessibility checking.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "date_time_util",
+    srcs = [
+        "DateTimeUtil.kt",
+    ],
+    manifest = "//utility/src/main/AndroidManifest.xml",
+    resource_files = ["//utility/src/main/res/values/strings.xml"],
```

Comment:
Try only "src/main/res/values/strings.xml"

Reply:
when I use `manifest = "src/main/AndroidManifest.xml"`

```bash
ERROR: /home/farees/opensource/oppia-android/utility/src/main/java/org/oppia/android/util/datetime/BUILD.bazel:7:19: Merging manifest for //utility/src/main/java/org/oppia/android/util/datetime:date_time_util_base failed: missing input file '//utility/src/main/java/org/oppia/android/util/datetime:src/main/AndroidManifest.xml'
ERROR: /home/farees/opensource/oppia-android/utility/src/main/java/org/oppia/android/util/datetime/BUILD.bazel:7:19 Merging manifest for //utility/src/main/java/org/oppia/android/util/datetime:date_time_util_base failed: 1 input file(s) do not exist
INFO: Elapsed time: 20.467s, Critical Path: 6.35s
INFO: 2 processes: 2 internal.
FAILED: Build did NOT complete successfully

```

Reply:
I'll take a closer look and get back to you by tomorrow.

Reply:
Without ``//`` isn't the relative path based on ``//utility/src/main/java/org/oppia/android/util``? The manifest & resource files aren't under that directory.

Reply:
> Without // isn't the relative path based on //utility/src/main/java/org/oppia/android/util? The manifest & resource files aren't under that directory.

I didn't get this. 
but for [AndroidManifest.xml](https://github.com/oppia/oppia-android/blob/develop/utility/src/main/AndroidManifest.xml) the relative path has to be `//utility/src/main/` right?

And I think we need to use `//` here as the AndroidManifest.xml is behind the util package

Just in case tried to build with `//utility/src/main/java/org/oppia/android/util` 
```bash
ERROR: /home/farees/opensource/oppia-android/utility/src/main/java/org/oppia/android/util/datetime/BUILD.bazel:7:19: no such package 'utility/src/main/java/org/oppia/android/util/AndroidManifest.xml': BUILD file not found in any of the following directories. Add a BUILD file to a directory to mark it as a package.
 - /home/farees/opensource/oppia-android/utility/src/main/java/org/oppia/android/util/AndroidManifest.xml and referenced by '//utility/src/main/java/org/oppia/android/util/datetime:date_time_util_base'
ERROR: Analysis of target '//utility/src/main/java/org/oppia/android/util/datetime:date_time_util_base' failed; build aborted: Analysis failed
INFO: Elapsed time: 8.203s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (17 packages loaded, 27 targets configured)

```